src/databricks/labs/ucx/contexts/application.py

```diff
@@ -358,17 +358,17 @@ def site_packages(self):
         return SitePackages([])
 
     @cached_property
-    def path_lookup(self):
+    def syspath_lookup(self):
         # TODO find a solution to enable a different cwd per job/task (maybe it's not necessary or possible?)
-        return PathLookup.from_sys_path(Path.cwd())
+        return SysPathLookup.from_sys_path(Path.cwd())
```

Comment:
```suggestion
        return PathLookup.from_sys_path(Path.cwd())
```
We use it for both notebooks and files, so it's not quite "sys.path"